src/wheel/bdist_wheel.py

```diff
@@ -131,7 +131,11 @@ def safer_version(version):
 
 
 def remove_readonly(func, path, excinfo):
-    print(str(excinfo[1]))
+    remove_readonly_exc(func, path, excinfo[1])
+
+
+def remove_readonly_exc(func, path, exc):
+    print(str(exc))
```

Comment:
This prints to stdout. Shouldn't it print to stderr instead?

Reply:
I'm not sure it should print anything at all. In all likelihood this was a leftover debugging line.